Formula/tcl-tk.rb

```diff
@@ -58,7 +73,7 @@ def install
 
         cd "unix" do
           system "./configure", *args
-          system "make", "TK_LIBRARY=#{lib}"
```

Comment:
Originally I thought that passing an explicit `TK_LIBRARY` might be what was causing the icon files to install to the lib. But what I found over the course of about a dozen test builds is that it makes no difference to any part of the build whether that argument is there or not.

Since it wasn't doing anything, and seems like a potential source of issues, I removed it.